### DISCUSSION TITLE
Allow adding beatmaps to collection on pause and fail menu

### DISCUSSION THREAD

**Discussion by cyperdark** [ppy/osu#29769] [Ideas]
Sometimes you cant reach to result screen and add beatmaps to collection, and we'll be cool to have it on pause and fail menu as well.

"But we can it in song select menu", no adding to collection in menu is garbage, you need to open dialog menu every time to add map to each collection, yes you can use dropdown menu "All beatmaps", but with tablet sometimes you click on collection and loses beatmap you wanted to add

**Comment by peppy** [comment]
This has been added to the results screen, might make sense to somehow add it to these screens too (if we can use a common method of display).
